### DISCUSSION TITLE
Is it an unsupported hack to used attibutes as multi-level tags?

### DISCUSSION THREAD

**Discussion by oshino29** [TriliumNext/Trilium#2736] [General]
I manage my code snippets in Trilium. my attributes are like
```
#snippets:go
#snippets:go:core
#snippets:go:web
#concept:go:unit_test

#snippets:js:vue
#issue:vscode:config
```
I found that when managing snippets in this way, attributes act rather flexible for limiting the scope of results in the search
(keywords like `:go` `go:core` `snippet:   vue` all give expected results in current 0.50.x version)

Is this an unsupoorted hack or an unmentioned feature? ( I did't found mention of such usage in [wiki - attributes](https://github.com/zadam/trilium/wiki/Attributes))

**Comment by zadam** [comment]
Trilium doesn't see it as a multilevel tags, it's just a normal tag with a name like `snippets:go:web`. It is supported in the sense that it should work in the future as well.

[Cloning notes](https://github.com/zadam/trilium/wiki/Cloning-notes) was envisioned for this use case, as it allows you to put a note into multiple places (so notes themselves become a tag hierarchy). But it might not suit everyone's needs ...
